doc/source/tutorial/tutorial_transforming.ipyml

```diff
@@ -30,66 +28,117 @@ cells:
     from larray import __version__
     __version__
 
-  id: 2
 
-- markdown: |
-    ## Manipulating axes
+- code: |
+    # load the 'demography_eurostat' dataset
+    demo_eurostat = load_example_data('demography_eurostat')
 
 
 - code: |
-    # let's start with
-    pop = load_example_data('demography').pop[2016, 'BruCap', 90:95]
+    # get a copy of the 'pop' array from the 'demography_eurostat' dataset
+    pop = demo_eurostat.pop.copy()
     pop
 
-  id: 3
+
+- markdown: |
+    ## Manipulating axes
+    
+    
+    LArray offers several to manipulate the axes and labels of an array:
+    - [set_labels](../_generated/larray.LArray.set_labels.rst#larray.LArray.set_labels): to replace all of some labels of one or several axes.
+    - [rename](../_generated/larray.LArray.rename.rst#larray.LArray.rename): to replace one of several axis names.
+    - [set_axes](../_generated/larray.LArray.set_axes.rst#larray.LArray.set_axes): to replace one of several axes.
+    - [transpose](../_generated/larray.LArray.transpose.rst#larray.LArray.transpose): to modify the order of axes.
+    - [drop](../_generated/larray.LArray.drop.rst#larray.LArray.drop): to remove one or several labels.
+    - [combine_axes](../_generated/larray.LArray.combine_axes.rst#larray.LArray.combine_axes): to combine axes.
+    - [split_axes](../_generated/larray.LArray.split_axes.rst#larray.LArray.split_axes): to split one or several axes.
+    - [reindex](../_generated/larray.LArray.reindex.rst#larray.LArray.reindex): to reorder, add and remove labels of one or several axes.
+    - [insert](../_generated/larray.LArray.insert.rst#larray.LArray.insert): to insert a label at a given position.
+
 
 - markdown: |
     ### Relabeling
     
-    Replace all labels of one axis
+    Replace all of some labels of an axis:
 
 
 - code: |
-    # returns a copy by default
-    pop_new_labels = pop.set_labels('sex', ['Men', 'Women'])
+    # replace all labels of the 'gender' axis by passing the list of all new labels
+    pop_new_labels = pop.set_labels('gender', ['Men', 'Women'])
     pop_new_labels
 
-  id: 4
 
 - code: |
-    # inplace flag avoids to create a copy
-    pop.set_labels('sex', ['M', 'F'], inplace=True)
+    # set all labels of the 'country' axis to uppercase by passing the function str.upper()
+    pop_new_labels = pop.set_labels('country', str.upper)
+    pop_new_labels
+
+
+- code: |
+    # replace only one label of the 'gender' axis by passing a dict
+    pop_new_labels = pop.set_labels('gender', {'Male': 'Men'})
+    pop_new_labels
+
+
+- code: |
+    pop2 = pop.copy()
+    
+    # by default, the 'set_labels' method returns a modified copy of the original array.
```

Comment:
it is important to say but... I wouldn't speak about inplace, because I am unsure we should support inplace operations at all and we support them for so few methods that it's a bit ridiculous at the moment.